etc/config/c++.amazon.properties

```diff
@@ -1102,6 +1102,33 @@ compiler.hexagonclang1605.exe=/opt/compiler-explorer/clang+llvm-16.0.5-cross-hex
 compiler.hexagonclang1605.compilerType=clang-hexagon
 compiler.hexagonclang1605.compilerCategories=clang-hexagon
 
+################################
+# Clang for Loongarch
+group.loongarch-clang.compilers=&loongarch64clang
+group.loongarch-clang.supportsBinary=true
+group.loongarch-clang.supportsExecute=false
+group.loongarch-clang.isSemVer=true
+group.loongarch-clang.licenseName=LLVM Apache 2
+group.loongarch-clang.licenseLink=https://github.com/llvm/llvm-project/blob/main/LICENSE.TXT
+group.loongarch-clang.licensePreamble=The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+
+group.loongarch64clang.compilers=loongarch64-clang:loongarch64-clang1701:loongarch64-clang1810
+group.loongarch64clang.options=-target loongarch64-unknown-elf --gcc-toolchain=/opt/compiler-explorer/loongarch64/gcc-13.2.0/loongarch64-unknown-linux-gnu --sysroot=/opt/compiler-explorer/loongarch64/gcc-13.2.0/loongarch64-unknown-linux-gnu/loongarch64-unknown-linux-gnu/sysroot
+group.loongarch64clang.objdumper=/opt/compiler-explorer/loongarch64/gcc-13.2.0/loongarch64-unknown-linux-gnu/bin/loongarch64-unknown-linux-gnu-objdump
+group.loongarch64clang.baseName=Loongarch64 clang
+group.loongarch64clang.groupName=Loongarch64 Clang
+group.loongarch64clang.compilerCategories=clang
+group.loongarch64clang.demangler=/opt/compiler-explorer/clang-trunk/bin/llvm-cxxfilt
+
+compiler.loongarch64-clang1701.exe=/opt/compiler-explorer/clang-17.0.1/bin/clang++
+compiler.loongarch64-clang1701.semver=17.0.1
+compiler.loongarch64-clang1810.exe=/opt/compiler-explorer/clang-18.1.0/bin/clang++
+compiler.loongarch64-clang1810.semver=18.1.0
+compiler.loongarch64-clang.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
+compiler.loongarch64-clang.semver=(trunk)
+compiler.loongarch64-clang.isNightly=true
+compiler.loongarch64-clang.alias=loongarch64clang
```

Comment:
alias is not needed here, this is only if that compiler id existed before

Reply:
Thanks, I will fix this.